files/image_config/platform/rc.local

```diff
@@ -351,6 +351,10 @@ if [ -f $FIRST_BOOT_FILE ]; then
         mv /host/grub.cfg /host/grub/grub.cfg
     fi
 
+    # If there is a platform dependant script, execute it
+    if [ -x /usr/share/sonic/device/${onie_platform}/platform_setup ]; then
+        /usr/share/sonic/device/${onie_platform}/platform_setup
```

Comment:
better to be $platform, not all platform are onie platform

Reply:
porting guide needs to be updated. https://github.com/Azure/SONiC/wiki/Porting-Guide

Reply:
I committed changes to use platform instead of device_platform.
Regarding the proposed changes in the wiki file Porting-Guide.md, how can I submit them?

Reply:
You should be able to open a pull request against the https://github.com/Azure/SONiC repo using the same procedure you used to generate this pull request.

Reply:
I do not see Porting-Guide.md when I clone git@github.com:Azure/SONiC.git

Reply:
Ah, yes, because it's part of the wiki, and GitHub treats the wiki differently than the repo itself. For now, you can open an issue in that repo and paste your proposed changes for the file in the description.